normits_demand/models/notem/production_models.py

```diff
@@ -180,6 +181,14 @@ def __init__(self,
             export_home=export_home,
             report_home=report_home,
         )
+        # Create a logger
+        logger_name = "%s.%s" % (__name__, self.__class__.__name__)
+        log_file_path = os.path.join(self.export_home, self._log_fname)
+        self._logger = nd.get_logger(
+            logger_name=logger_name,
+            log_file_path=log_file_path,
+            instantiate_msg="Initialised new NoTEM Logger",
```

Comment:
Change this message to be model specific - HB Production model

Reply:
Updated